xml/System/Environment+SpecialFolder.xml

```diff
@@ -611,7 +611,7 @@
       </ReturnValue>
       <MemberValue>24</MemberValue>
       <Docs>
-        <summary>The file system directory that contains the programs that appear in the **Startup** folder for all users. This special folder is valid only for Windows NT systems. Added in the .NET Framework 4.</summary>
+        <summary>The file system directory that contains the programs that appear in the **Startup** folder for all users. This special folder is valid only for Windows NT systems.</summary>
```

Comment:
[We still support this](https://github.com/dotnet/runtime/search?q=extension%3Acs+CommonStartup):

```suggestion
        <summary>The file system directory that contains the programs that appear in the **Startup** folder for all users.</summary>
```